openpathsampling/pathmover.py

```diff
@@ -320,11 +333,8 @@ def move(self, globalstate):
 
         return paths.EmptyPathMoveChange()  # pragma: no cover
 
-    def __str__(self):
-        if self.name == self.__class__.__name__:
-            return self.__repr__()
-        else:
-            return self.name
+    def __repr__(self):
+        return self.name + '(%s)' % hex(id(self))[2:]
```

Comment:
You are right. I think I tried to have something that is short but kind of unique and does not depend on anything that storage uses.